pkgs/by-name/wi/wildcard/package.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, blueprint-compiler
+, cargo
+, desktop-file-utils
+, fetchFromGitLab
+, libadwaita
+, meson
+, ninja
+, pkg-config
+, rustPlatform
+, rustc
+, stdenv
+, wrapGAppsHook4
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "wildcard";
+  version = "0.3.3";
+
+  src = fetchFromGitLab {
+    domain = "gitlab.gnome.org";
+    owner = "World";
+    repo = "Wildcard";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-jOv0l1vnfDePWF7SAbsBFipPAONliPdc47xj79BJ+rc=";
+  };
+
+  cargoDeps = rustPlatform.fetchCargoTarball {
+    inherit (finalAttrs) src;
+    hash = "sha256-RWEUS7WA/mKCjHLUvkdzsHGg9xQrEVGnvqncBTy1U8c=";
+  };
+
+  nativeBuildInputs = [
+    blueprint-compiler
+    cargo
+    desktop-file-utils
+    libadwaita
+    meson
+    ninja
+    pkg-config
+    rustPlatform.cargoSetupHook
+    rustc
+    wrapGAppsHook4
+  ];
+
+  meta = {
+    description = "Test your regular expressions";
+    longDescription = ''
+      Wildcard gives you a nice and simple to use interface to test/practice regular expressions.
+    '';
+    homepage = "https://gitlab.gnome.org/World/Wildcard";
+    downloadPage = "https://gitlab.gnome.org/World/Wildcard/-/releases/v${finalAttrs.version}";
+    license = lib.licenses.gpl3Plus;
+    maintainers = with lib.maintainers; [];
```

Comment:
You don't want to add yourself to maintainers? (See `maintainers/maintainer-list.nix`) first

Reply:
I've only began learning nix and packaging, so I think I don't have enough experience to actually maintain a package. So, no, I don't want to add myself to maintainers (not now at least).

Reply:
But you are still submitting it, how do you think you cannot maintain it?

Reply:
I think maintaining a package is a big responsibility that requires experience and involvement. I don't think I am yet ready for this it: I don't have enough experience with packaging in nix and I don't yet know if I am interested in this kind of stuff. So I want to leave maintaining at more capable hands for now.